### DISCUSSION TITLE
ControlNet Model

### DISCUSSION THREAD

**Discussion by Elevenheights** [TencentARC/PhotoMaker#12] [Ideas]
Any chance there will be a controlnet model of this coming?

**Comment by Paper99** [comment]
Hi all, ControlNet now is supported.
Besides, we update the PhotoMaker V2 (with improved ID fiedilty). Read [the report](https://github.com/TencentARC/PhotoMaker/blob/main/README_pmv2.md) for details.

**Comment by smithian123** [comment]
Can you use controlnet with the Pinokio interface?
